package.json

```diff
@@ -9,7 +9,7 @@
     "electron-store": "^8.0.0",
     "fs": "0.0.1-security",
     "keytar": "^7.7.0",
-    "kill-port": "^2.0.1",
+    "-port": "^2.0.1",
```

Comment:
Probably a typo?

Reply:
Yes, sorry must have happened when uncommenting this part